### ISSUE TITLE
Convert SQL Server 2014 DB/code to use SQL Lite

### ISSUE THREAD

**Issue by CodeLiftSleep** [CodeLiftSleep/Goal-Line-Stand-Football#31]
End users are most likely not going to have SQL installed so it's unrealistic to use a DB that would require that. SQL Lite is an embedded DB which does not require and end user setup, and still supports LINQ and EF.

**Comment by CodeLiftSleep** [comment]
Almost complete. SQLite doesn't have an Update Table function, had to write a Bulk Insert function as a single transaction to as closely mimic it as I could. Also have to explicitly declare the connection string, returning it as a function throws an error, and am completing the various field changes that need to happen for it to be finished. Mainly that each string must be 'James' and not James, or it throws an error.

Fun stuff!

**Comment by CodeLiftSleep** [comment]
Completed.

Added a DROP TABLE Function to completely remove the Datatable from the DB
Added a DELETE TABLE Function to remove all records from the DataTable without removing it from the DB

Speed stayed roughly the same, about 1:16 seconds for all records to generate completely from start to finish and be updated to the DB.

Project is now Properly Built(Thanks Diogo!) with dependencies, I removed all references to SQL Server from the project, and added the SQLite DB as a Local DB in the debug folder, which should work for everyone.
